src/dmd/dsymbolsem.d

```diff
@@ -3807,7 +3807,7 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
         }
 
         sc.stc &= ~STC.static_; // not a static constructor
-        sc.flags |= SCOPE.ctor;
+//        sc.flags |= SCOPE.ctor;
```

Comment:
Don't comment out code that isn't printf in git.

Reply:
I know, that's why this is labelled WIP. See also https://github.com/dlang/dmd/pull/12932